File: java/com/example/Eco/data/ActiviteRecyclage.kt
package com.example.Eco.data

data class ActiviteRecyclage(
    val _id: String? = null,
    val dateAndTime: String,
    val quantity: Int,
    val recyclableMaterial: String,
    val user: String,
    val image: String // Ajout d'un champ pour l'image (URL de l'image)
)

File: Models/ActiviteRecyclage.swift
import Foundation

struct ActiviteRecyclage: Identifiable, Codable {
    var id: String? // Le champ _id en Kotlin devient id en Swift
    var dateAndTime: String
    var quantity: Int
    var recyclableMaterial: String
    var user: String
    var image: String // URL de l'image
    
    enum CodingKeys: String, CodingKey {
        case id = "_id"
        case dateAndTime
        case quantity
        case recyclableMaterial
        case user
        case image
    }
}
